### ISSUE TITLE
Localization Kodi v.19

### ISSUE THREAD

**Issue by arustler** [RobLoach/lutris-kodi-addon#39]
In Kodi v.19 the Addon isn't working anymore.
This is due to the change of the localization structure [.](https://kodi.wiki/view/Language_support)

```
Error Contents: not all arguments converted during string formatting
    Traceback (most recent call last):
    File "/home/kodi/.kodi/addons/script.lutris/default.py", line 121, in <module>
    commands.append((language(30200) % (runner), 'RunPlugin(%s?mode=folder&id=%d&slug=%s&gamename=%s)' % (base_url, game_id, slug + ' --reinstall', name)))
    TypeError: not all arguments converted during string formatting
    -->End of Python script error report<--
```

**Comment by arustler** [comment]
I've updated the resources folder and changed the string formating in to comply to python versions greater 2.7
Created pull request #40

**Comment by solbero** [comment]
This is solved with merging PR [#40](https://github.com/RobLoach/lutris-kodi-addon/pull/40)
